.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,71 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master, feature/* ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+  schedule:
+    - cron: '33 19 * * 6'
```

Comment:
How often does this run?  I'm not up on my cron syntax.

Reply:
Ah yeah, I had to look this up. This value was auto-added by the config generator that GitHub provides by default.

The docs are here:
https://docs.github.com/en/actions/reference/events-that-trigger-workflows

Here's a pertinent snippet:
```
┌───────────── minute (0 - 59)
│ ┌───────────── hour (0 - 23)
│ │ ┌───────────── day of the month (1 - 31)
│ │ │ ┌───────────── month (1 - 12 or JAN-DEC)
│ │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
│ │ │ │ │                                   
│ │ │ │ │
│ │ │ │ │
* * * * *
```

`33 19 * * 6` means 7:33pm every Saturday.

There are more complex rules than just specifying specific times - more details in the linked docs above.

This shedule trigger exists for this specific workflow because the security flaws it detects might change even if the code doesn't.